apps/gateway/src/openApi/schemas/ai.ts

```diff
@@ -0,0 +1,98 @@
+import { z } from '@hono/zod-openapi'
+import { ChainEventTypes, StreamEventTypes } from '@latitude-data/constants'
+import { messageSchema } from '@latitude-data/core/browser'
+
+export const languageModelUsageSchema = z.object({
+  completionTokens: z.number().optional(),
+  promptTokens: z.number().optional(),
+  totalTokens: z.number().optional(),
+})
+
+export const toolCallSchema = z.object({
+  id: z.string(),
+  name: z.string(),
+  arguments: z.record(z.any()),
+})
+
+export const configSchema = z.object({}).passthrough()
+export const providerLogSchema = z.object({}).passthrough()
```

Comment:
no schema for provider log?

Reply:
It has not really been used in the API afaik